spartan/aztec-network/files/config/get-private-key.sh

```diff
@@ -2,7 +2,7 @@
 set -eu
 
 # We get the index in the config map from the pod name, which will have the service index within it
-KEY_INDEX=$(echo $POD_NAME | awk -F'-' '{print $NF}')
```

Comment:
K8S_POD_NAME already existed on containers, POD_NAME was the name given to the same env var in the init containers